### ISSUE TITLE
Exception in other datasets during network training

### ISSUE THREAD

**Issue by zhenbangW** [Momuli/SMENet#7]
I hope to use the network model you designed to train the new dataset I created (3 categories). I did not modify the image size of the input network (using the initial 400Pixel in the code), but only modified the file used to create the dataset, voc2017.py. The final training loss was 0.899 (different learning rates were attempted multiple times), but the eval result was poor, with a mAP of only 0.0566. Excuse me, are there any areas where I haven't made the necessary modifications or omissions?
The visualization results of target detection are shown in the following figure. For the categories detected in the red box, they are correct, while others are error detection
![PW3QXO`83{_{5WBK)4S({TY](https://github.com/Momuli/SMENet/assets/44695439/2d042a38-e687-4f6d-9810-670a1857fd3f)
I just rewrote the__ getitem__ ()
![image](https://github.com/Momuli/SMENet/assets/44695439/71e5d0d7-e43e-46a5-944c-6f9ca2eaade9)
